doc/logging.md

```diff
@@ -0,0 +1,19 @@
+# How does Happy test log work
+
+## Types of Happy logs
+
+  * Debug logs - Logs related to Happy's role as test coordinator and orchestrator, where it facilitates the activities of virtual device nodes.
+
+  * Device logs - Logs related to software such as openweave running on individual virtual device nodes within a Happy test network. OpenWeave is one of the software example, device logs may have OpenWeave logs.
```

Comment:
We can clean it up in a subsequent pass; however, the capitalization of OpenWeave here is inconsistent.

Reply:
good eyes:)
I fixed it to "OpenWeave"